 <launch>
    <arg name="device" default="/dev/video0" />

    <node name="example_node" pkg="uvc_camera" type="uvc_camera_node" output="screen">
      <remap from="image_raw/compressed" to="/uvc_camera/image_raw/compressed"/>
      <param name="device" value="$(arg device)" />
      <param name="width" value="2560" />
      <param name="height" value="960" />
      <param name="format" value="jpeg" />
      <param name="fps" value="60" />
      <param name="frame" value="wide_stereo" />
      <param name="power_line_frequency" value="50"/>
      <param name="auto_exposure" value="1" />
      <param name="exposure_absolute" value="75" />
      <param name="exposure_auto_priority" value="0" />
      <param name="auto_white_balance" value="1" />
      <param name="white_balance_temperature" value="6500" />
      <param name="brightness" value="30" />
      <param name="auto_focus" value="0" />
      <param name="focus_absolute" value="0" />
      <param name="contrast" value="4" />
      <param name="saturation" value="80" />
      <param name="hue" value="0" />
      <param name="gamma" value="100" />
      <param name="sharpness" value="3" />
      <param name="backlight_compensation" value="2" />
      <param name="gain" value="30" />
      <param name="auto_gain" value="0" />
      <param name="horizontal_flip" value="0" />
      <param name="vertical_flip" value="0" />
    </node>

    <node name="image_transport1" pkg="image_transport" type="republish" args="compressed in:=/uvc_camera/image_raw theora out:=/uvc_camera/image_raw"/>
    <!-- <node name="image_transport2" pkg="image_transport" type="republish" args="compressed in:=/uvc_camera/image_raw raw out:=/uvc_camera/image_raw"/> -->
</launch>
